src/implementations/twig/storybook/.storybook/manager-head.html

```diff
@@ -51,3 +51,14 @@
   type="text/javascript"
   defer
 ></script>
+<style>
```

Comment:
I'd suggest we avoid making this change as it's not really reflecting themes, but css resources.
Have you checked https://github.com/tonai/storybook-addon-themes? see https://github.com/tonai/storybook-addon-themes/issues/21 which will probably be more relevant in our html case

Reply:
i'm confused, before you said that cssresources is the right "title" for this tab, here it seems you are suggesting to use a different approach regarding the theme switcher. Personally speaking i only spent a bit of time in the past looking for alternatives to the current approach we are using and i haven't found much, i'm not surely enthusiast about the UI provided by cssresources nor the user experience that we will offer. 

As far as we are using cssresources, though, it looks to me that its label "css resources" doesn't really offer a clear indication of the relevance it would have in our system, therefore here we only perform its replacement into a more informative "ECL themes", since we are going to have a theme based system and this would be the interface the user will have to switch between them. 

If there is no agreement even about the opportunity to rename this tab, i don't mind, let's close this, If you believe there is a better overall approach for a  "theme switcher" in storybook i would be glad to entirely replace cssresources, indeed.

Reply:
Summarizing my previous points to clear up confusion:
- changes in the pull request are not fully ok for me
- my suggestion is to leave the name of the tab to css resources (revert css change)
- the other updates seem fine
- i suggested another addon for solving what you're actually trying to solve: expose "themes" feature